public/scripts/index.js

```diff
@@ -1,13 +1,72 @@
-(function (){
+function getWordFormData(form) {
+    let wordData = {}
+    //get content, reading example
+    wordData.content = document.getElementById('word-content').value
+    wordData.phoneticReading = document.getElementById('word-reading').value
+    wordData.example = document.getElementById('word-example').value
+    //get meanings
+    wordData.meanings = Array.from(document.getElementById('word-meanings').querySelectorAll('input')).map((word) => { return word.value })
+
+    //get knowledge level
+    let reducer = (accumulator, currentValue) => {
+        if (currentValue.checked) {
+            return accumulator + parseInt(currentValue.value)
+        } else { return accumulator }
+    }
+    wordData.knowledgeLevel = Array.from(document.getElementById('word-knowledge-rate').querySelectorAll('input')).reduce(reducer, 0)
+    if (wordData == 0)
```

Comment:
Use brackets: `if () {}`.

Reply:
Also, `wordData` is an objet, not a number, right?
Did you wanted to check `wordData.knowledgeLevel`?